ee/modules/500-operator-trivy/openapi/config-values.yaml

```diff
@@ -13,6 +13,17 @@ properties:
       **Warning.** Specifying a value different from the one currently used (in the existing PVC) will result in disk re-provisioning and all data will be deleted.
       
       If `false` is specified, `emptyDir` will be forced to be used.
+  additionalVulnerabilityReportFields:
+    type: array
+    description: |
+      A list of additional fields from the vulnerability database to add to the VulnerabilityReport.
+
+    items:
+      type: string
+    x-example:
```

Comment:
And maybe better use `x-examples`? I've saw in repository and we are using `x-examples` everywhere

Reply:
fixed